client/signup/steps/design-picker/index.jsx

```diff
@@ -54,7 +54,14 @@ class DesignPickerStep extends Component {
 
 	renderDesignPicker() {
 		// props.locale obtained via `localize` HoC
-		return <DesignPicker locale={ this.props.locale } onSelect={ this.pickDesign } />;
+		return (
+			<DesignPicker
+				theme="dark"
+				isGridMinimal
```

Comment:
isn't this flag supposed to be used only for AnchorFM (and maybe FSE) flows?

cc: @ciampo

Reply:
Yes it is. This PR was before yours that fixed the width. This was a workaround but now it's no longer needed!